.github/workflows/build-huggingface.yml

```diff
@@ -6,7 +6,7 @@ on:
       tgi-version:
         description: 'tgi version'
         required: true
-        default: '1.0.3'
```

Comment:
non-blocking: could you please add a runbook for this process so that others on the team can help with this type of work?

Reply:
Sure thing. Thanks